_includes/accessibility.html

```diff
@@ -1,18 +1,28 @@
 <section class="usa-prose site-prose margin-top-5">
-  <h2 id="general-accessibility-guidance-for-forms" class="usa-heading heading-margin-alt">General accessibility guidance for forms</h2>
+  <h2 id="general-accessibility-guidance-for-forms" class="usa-heading heading-margin-alt">Accessibility guidance</h2>
   <ul class="usa-content-list">
-    <li><strong>Customize accessibly.</strong> As you customize these templates, make sure they meet the accessibility guidelines in this introduction and as described for each control.</li>
     <li><strong>Don’t control element order with CSS.</strong> Display form controls in the same order in the HTML as they appear on screen. Don’t use CSS to rearrange the form controls. Screen readers narrate form elements in the order they appear in the HTML.</li>
     <li><strong>Align validation with inputs.</strong> Visually align validation messages with the input fields so people using screen magnifiers can read them quickly.</li>
     <li><strong>Use proper markup.</strong> Group each set of thematically related controls in a fieldset element. Use the legend element to offer a label within each one. The fieldset and legend elements make it easier for those who use screen readers to navigate the form.</li>
     <li><strong>Use legends.</strong> Use a single legend for fieldset (this is required). One common use of the fieldset and legend elements is a question with radio-button options for answers. The question text and radio buttons are wrapped in a fieldset, with the question itself being inside the legend tag.</li>
     <li><strong>Embed multiple fieldsets and legends for more complex forms.</strong></li>
     <li><strong>Use simple vertical layouts.</strong> Keep your form blocks in a vertical pattern. This approach is ideal, from an accessibility standpoint, because of limited vision that makes it hard to scan from right to left.</li>
   </ul>
-
-  <div class="site-note"><strong>Note:</strong> These components have been designed to support a wide range of screen readers, but they may not work with all versions.</div>
+  <h2 id="general-accessibility-guidance-for-forms" class="usa-heading heading-margin-alt">Usability guidance</h2>
```

Comment:
Duplicate ID to the one above. We can add a unique one like `general-usability-guidance-for-forms`. Example below. We can also simplify by removing the classes.

```suggestion
  <h2 id="general-usability-guidance-for-forms">Usability guidance</h2>
```